src/components/filter-selector/FilterSelector.tsx

```diff
@@ -76,16 +81,20 @@ const FilterSelector = <T extends string>({
 
   const filteredItems = useMemo(
     () =>
-      response.filter((item) =>
-        item.toLowerCase().includes(inputValue.toLowerCase())
+      response.filter((item: ObjectType) =>
+        item.name.toLowerCase().includes(inputValue.toLowerCase())
```

Comment:
what if in other place it can be title, do you really need to get an object with name and id, if yes better refactor this like in AsynAutoComplete

Reply:
that was fixed